test/framework/config.py

```diff
@@ -681,8 +682,29 @@ def test_update_build_option(self):
         """Test updating of a build option."""
         self.assertEqual(build_option('banned_linked_shared_libs'), None)
 
-        update_build_option('banned_linked_shared_libs', '/usr/lib64/libssl.so.1.1')
+        old_val = update_build_option('banned_linked_shared_libs', '/usr/lib64/libssl.so.1.1')
         self.assertEqual(build_option('banned_linked_shared_libs'), '/usr/lib64/libssl.so.1.1')
+        self.assertEqual(old_val, None)
+
+    def test_update_build_options(self):
+        """Test updating of a dictionary of build options."""
+        self.assertEqual(build_option('banned_linked_shared_libs'), None)
+        self.assertEqual(build_option('filter_env_vars'), None)
```

Comment:
Also use `cleanup_builddir` and `pr_target_account` here to test `update_build_options`?

Reply:
Sure, why not :) done!